src/animated.tsx

```diff
@@ -137,7 +137,7 @@ export const KeyboardProvider = ({
             },
           },
         ],
-        { useNativeDriver: true },
+        { useNativeDriver: Platform.OS === "web" ? false : true },
```

Comment:
Maybe could be simplified to `useNativeDriver: Platform.OS !== "web" `?
And add a comment above that we ignore web, because `useNativeDriver` is not supported there and just throws warnings?

Reply:
sure let me do that